tokio-console/src/view/resource.rs

```diff
@@ -21,7 +21,7 @@ use std::{cell::RefCell, rc::Rc};
 
 pub(crate) struct ResourceView {
     resource: Rc<RefCell<Resource>>,
-    async_ops_table: TableListState<AsyncOpsTable, 9>,
+    pub async_ops_table: TableListState<AsyncOpsTable, 9>,
```

Comment:
Let's make this `pub(crate)` to match the struct it's in.

```suggestion
    pub(crate) async_ops_table: TableListState<AsyncOpsTable, 9>,
```